tensorflow/python/ops/nn_impl.py

```diff
@@ -856,7 +856,9 @@ def depthwise_conv2d_v2(input,
     filter: 4-D with shape
       `[filter_height, filter_width, in_channels, channel_multiplier]`.
     strides: 1-D of size 4.  The stride of the sliding window for each
-      dimension of `input`.
+      dimension of `input`. Must have `strides[0] = strides[3] = 1`, and for
```

Comment:
This is already described above in the details section.  Do you really think it's necessary to have here too?